src/scene_file.cpp

```diff
@@ -0,0 +1,7 @@
+#include "scene_file.h"
+
```

Comment:
I'm not sure it is right...
`stream::get` documentation: https://www.cplusplus.com/reference/istream/istream/get/